pkg/rpcclient/waiter/waiter_test.go

```diff
@@ -38,16 +99,12 @@ func (c *AwaitableRPCClient) ReceiveExecutions(flt *neorpc.ExecutionFilter, rcvr
 func (c *AwaitableRPCClient) Unsubscribe(id string) error { return nil }
 
 func TestNewWaiter(t *testing.T) {
-	w := NewWaiter((RPCActor)(nil), nil)
-	_, ok := w.(NullWaiter)
-	require.True(t, ok)
```

Comment:
Just provide any other item to this constructor, no need to move this test to Actor.